### ISSUE TITLE
When "No Index File", folder links should point to “folder/" rather than "folder"

### ISSUE THREAD

**Issue by thierrymichel** [bdkjones/CodeKit2#391]
![ck-no-index-file](https://cloud.githubusercontent.com/assets/806883/3567858/66e25010-0b24-11e4-8b98-178901ea6e6b.png)

**Comment by bdkjones** [comment]
I don't quite understand. You're saying that you want the trailing slash appended to the URL?

**Comment by thierrymichel** [comment]
Yes.

All relatives paths (CSS, JS) are incorrect.
They point to the root, not to the related folder...

It's just a report, not a feature request.
I have just found it in a very particular case (working on a sass lib
project with a demo folder into my git repo...)

Thierry

**Comment by bdkjones** [comment]
Fixed for 2.1.9. Folders will now have trailing slashes in their `href` urls.

However, CodeKit does not enforce the trailing slash for your own URLs. So, if you write a link like this:

```
<a href='/someFolder/anotherFolder'>myLink</a>
```

The request to CodeKit will not include a trailing slash and CodeKit will not redirect your browser to:

```
http://name.local:5757/someFolder/anotherFolder/
```

But will load the URI:

```
http://name.local:5757/someFolder/anotherFolder
```

Which may break your relative links, etc. Apache, by default, does redirect to the trailing-slash URI (although this can be turned off). There are some reasons you may not want Apache to do that, however, and since CodeKit is a testing environment for developers, my expectation is that you specify exactly the URI you want CodeKit to load and CodeKit does not make any assumptions on your behalf.
